src/scrap.py

```diff
@@ -0,0 +1,262 @@
+import requests
+import os
+import discord
+from typing import List
+from bs4 import BeautifulSoup
+from dotenv import load_dotenv
+import sqlite3
+
+# For using my .ENV files
+load_dotenv()
+
+
+# -------------------FOR SQLLITE----------------------------
+
+# https://docs.python.org/3/library/sqlite3.html
+# https://www.blog.pythonlibrary.org/2021/09/30/sqlite/#:~:text=Here%20is%20how%20you%20would%20create%20a%20SQLite,the%20sqlite3%20module%20will%20create%20an%20empty%20database.
+
+
+def create():
+    # Creates the DB. Only needs to be called once.
+
+    connection = sqlite3.connect("jobs.db")
```

Comment:
suggest putting jobs.db in a variable or even an environment var if you're gonna repeat it everywhere in the code